### ISSUE TITLE
Error in Widget Builder issue

### ISSUE THREAD

**Issue by ramdev1980** [percussion/percussioncms#774]
**Describe the bug**
Using wave extension, we are getting 1 'missing form label' issues. When we are inspecting them, and fixing them with in jsp file by adding screenReaderLabel: 'Widget Source Code Editor', then also it is not getting fixed.

**To Reproduce**
1. Go to Widget Builder (http://localhost:9992/cm/app/?view=widgetbuilder&site=test&path=%2FSites%2Ftest%2F&debug=true&pathType=page)
2. Click on 'Create New Widget' icon on top right corner.
3. Run wave firefox extension.
4. See the error.

**Expected behavior**
There should be no error.

**Additional context**
It seems that the CodeMirror js library is not updated against the code written in jsp. Need to upgrade to version 5
